app/models/account.rb

```diff
@@ -28,8 +28,8 @@ class Account < ApplicationRecord
   # Required for configuring mixed in ActiveRecordCacheAside module.
   # Redis settings for ttl and namespacing reside in config/redis.yml
   #
-  redis REDIS_CONFIG['user_account_details']['namespace']
-  redis_ttl REDIS_CONFIG['user_account_details']['each_ttl']
+  redis REDIS_CONFIG[:user_account_details][:namespace]
+  redis_ttl REDIS_CONFIG[:user_account_details][:each_ttl]
```

Comment:
Same comment here WRT moving this into a kind of "Rails 6 preparation PR".

Reply:
These changes are because of deprecation warnings.  So if we were to split them out, would it make more sense to be _after_ the Rails 6 change?

Reply:
Oh interesting. Ya, that could work too. 

Not a big deal either way, maybe just a nice-to-have WRT smaller PR's.